iati/guidance_and_support/models.py

```diff
@@ -98,3 +99,22 @@ class KnowledgebasePage(AbstractContentPage):
     """A base for a single Knowledgebase page."""
 
     subpage_types = []
+
+
+class CommunityPage(DefaultPageHeaderImageMixin, AbstractIndexPage):
+    """A base for the Community page."""
+
+    parent_page_types = ['home.HomePage']
+    subpage_types = []
+
+    text_box = models.TextField(max_length=255, null=True, blank=True, help_text='A small ammount of text describing the community page.')
+
+    button_link_text = models.CharField(max_length=255, null=True, blank=True, help_text='The text to appear on the button of the community page.')
```

Comment:
Please could we add an `URLField` for the Discuss link as `button_link`? The URL for discuss can be added by admin. Would be good if this was dynamic versus static.